_tutorials/npm.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Origami via npm
+description: A guide on how to use Origami via npm.
+cta: Learn how to Origami via npm
+---
+
+# {{page.title}}
+
+
+If migrating from Bower to npm, [please refer to our migration guide](https://origami.ft.com/docs/tutorials/bower-to-npm/).
+
+## Benefits
+
+- Tooling such as Dependabot/Renovate/Greenkeeper/Snyk will work for the Origami dependencies you have.
+
+- Other tooling which by default only work with npm dependencies will now work without requiring any changes. I.E. Webpack.
+
+## Drawbacks
+
+- The Origami Team have not finalised how Origami components on npm should work, it is considered not stable.
+
+- Need to use extra tooling to ensure multiple versions of Origami are not included in the dependency tree.
+
+- Documentation for Origami assumes the Bower interface and not the npm interface. I.E. The package-names don't start with `@financial-times`.
+
+
+## How to avoid multiple versions of Origami in the dependency tree
+
+We have built a tool called [lockspot](https://www.npmjs.com/package/lockspot) which can be used to report about any dependencies which have multiple versions of themselves, such as Origami dependencies. 
```

Comment:
I find this sentence a bit confusing to read. specifically "multiple versions of themselves". 

I'm also a little concerned about the language " we have built a tool" for something I did in my private time in a personal capacity

Reply:
@chee how about this?

```suggestion
There is a tool called [lockspot](https://www.npmjs.com/package/lockspot) which can be used to report about any packages which are included with more than one version, such as Origami dependencies. E.G. Detecting that `o-table@6` and `o-table@7` are both in the dependency tree.
```

Reply:
that's great !!!